README.md

```diff
@@ -54,6 +54,85 @@ Alternatively, the registry data can be accessed in XML format at the following
 
 These XML pages are compatible with the XML format once provided by `myosg.grid.iu.edu`.
 
+### XML page arguments
+
+Some of the XML pages listed above accept arguments for filtering queries.
+These are compatible with the arguments once used by `myosg.grid.iu.edu`.
+
+#### For the Resource Topology (rgsummary) and Resource Downtime (rgdowntime) pages
+
+Boolean filters:
+
+| The following argument(s)... | Will have the effect of...                         |
+|------------------------------|----------------------------------------------------|
+| `active=on&active_value=0`   | showing only inactive resources                    |
+| `active=on&active_value=1`   | showing only active resources                      |
+| `disable=on&disable_value=0` | showing only disabled resources                    |
+| `disable=on&disable_value=1` | showing only enabled resources                     |
```

Comment:
It seems like maybe /we/ got this wrong. Reading this over, I would expect `disable_value=1` to correspond to showing only disabled resources.

Reply:
Nope. I checked their source; if you don't have `disable=on`, `disable_value` does nothing.  Which is good because of the number of people that have `active=on&active_value=1&disable_value=1` in their queries, which would show resources that are both "active" and "disabled" which is nonsensical.

Reply:
Ah ok, I was confused by our earlier conversation, then. Yikes.